docs/website/contents/for-developers/PreflightGuide.md

```diff
@@ -0,0 +1,101 @@
+# Preflight guide
+
+This document lists possible steps in getting familiar with the core of the Cardano universe with a focus on topics that are important for the daily life of a Consensus developer.
+
+Every step has a clear **goal** as well as optional ideas for other activities.
+
+This document contains various footnotes with additional information that can be safely skipped on a first pass.
+
+## Setting up a local node
+
+The [cardano-node][] (usually just called "node") is the most important consumer[^other-consensus-consumers] of the Consensus and Storage layer maintained by the Consensus team.
+It is *the* way to get a full copy of "the" blockchain for any given network[^cardano-node-purpose], in particular and most importantly, for **mainnet**. This process is called "syncing".
+
+Having a local copy of the node is not absolutely essential for your daily work, but it can be useful in several cases:
+
+ - Assessing a change for correctness and performance problems by using the entire multi-year history of Cardano as a test case.
+ - Quickly gathering empirical statistics of quantities of interest, like minimum/average/maximum size/validation time/frequency of blocks.
+ - Direct access to any on-chain data, like the current stake distribution or UTxO size.
+
+### Disclaimer
+
+ - The node is currently relatively resource-intensive[^resource-relative], see the minimum system requirements [node release page][].
+    - It uses a lot of memory, and the memory requirements are growing as the chain evolves[^utxo-hd].
+    - It inherently requires a lot of disk space (as of January 2024, 150GB).
+    - During syncing, it is quite CPU-intensive (albeit mostly only on a single core).
+      Once syncing is done, it is very light on CPU usage.
+ - You do not need to run the node at all times.
+   The stored blockchain can also be consumed by other developer tools that are independent of the node.
+ - Syncing can take multiple days (full sync in December 2023 on a decently powerful server: 2 days).
+   Unless your network connection is very slow compared to your CPU, the bottleneck is likely your CPU.
+ - It is safe to cleanly interrupt syncing, eg via <kbd>Ctrl</kbd> + <kbd>C</kbd>.
+   The node is able to gracefully resume on restart.
+
```

Comment:
```suggestion
- If you're in a rush and do not mind a little extra clutter, it's possible to sync to mainnet in about two hours by using the [Mithril client](https://mithril.network/doc/). See [here](https://mithril.network/doc/manual/getting-started/bootstrap-cardano-node/) for instructions.

```

Reply:
Thanks, I incorporated this in a section called "Syncing faster" :+1: